docs/submitting.md

```diff
@@ -314,3 +314,9 @@ After submission:
 - The review issue will be closed, and an automatic tweet from [@JOSS_TheOJ](https://twitter.com/JOSS_TheOJ) will announce it!
 
 If you want to learn more details about the review process, take a look at the [reviewer guidelines](reviewer_guidelines.html).
+
+## Confidential requests
+
+Please write admin@theoj.org with confidential matters such as retraction requests, report of misconduct, and retroactive author name changes.
+In case of a name change (e.g., after an author transitions to their true gender identity, marriage, divorce, or religious conversion), the DOI will be unchanged and the paper will be updated to use the new name and note that a name has been changed (but without identifying the author).
```

Comment:
rather than all the details of why one's name might change, perhaps this could just be "In the case of a legal name change, ..."

Reply:
I'm fine with that and will push a change if I don't hear otherwise.